store/dataStore.ts

```diff
@@ -18,6 +40,19 @@ const initialState: IDataStoreState = {
     id: '',
     email: '',
   },
+  NFLTeams: {
```

Comment:
m: Same here. Alphabetize the property names.

Reply:
Reordered the property names in alphabetical order.